api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -146,6 +146,9 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
 
   @Override
   public ResponseEntity<Workspace> createWorkspace(Workspace workspace) {
+    if (workspace.getName().equals("")) {
```

Comment:
Should this be trimmed before comparison? Alternatively, ClojureScript has `blank?` which returns, "True if s is nil, empty, or contains only whitespace," which might be useful to us.

Reply:
Considering that I am adding a utils file in the other PR I have open, that could be a good place to put that. I can throw something in there. (After I merge that one in)